src/lib/mina_base/permissions.ml

```diff
@@ -552,7 +650,14 @@ let%test_unit "json value" =
         send: "Signature",
         receive: "None",
         setDelegate: "Signature",
-        setPermissions: "Signature",
+        setPermissions: { 
+          "auth": "Signature",
+          "version": { 
```

Comment:
Should these values be taken from the compile-time configuration, rather than duplicating them here?